mitodl/ocw-hugo-themes
# refactor: remove youtube shortcode

#### Pre-Flight checklist


#### What are the relevant tickets?
closes #1060 

#### What's this PR do?
This PR removes the [{{< youtube >}} shortcode](https://github.com/mitodl/ocw-hugo-themes/blob/main/course-v2/layouts/shortcodes/youtube.html) which is overriding the builtin Hugo {{< youtube >}} shortcode from course-v2 theme.

#### How should this be manually tested?
One of the ways to test this would be to have the {{< youtube >}} shortcode inside a content markdown file. When we know that our custom videojs-youtube player does not appear, but in fact the default Youtube embedded player appears, we will know that our implemented change is working.

Sample instructions:
1. Checkout to this branch
2. Create an empty directory inside course-v2/content/. Let's name it "youtube" i.e course-v2/content/youtube for the sake of this example.
3. Make a markdown file inside course-v2/content/youtube/. Let's name it "_index.md" i.e course-v2/content/youtube/_index.md
4. Inside the newly created _index.md place this:  
```
---
---
{{< youtube dQw4w9WgXcQ >}}
```
i.e, we have empty front matter, and inside the content we are using the youtube shortcode with the video ID of dQw4w9WgXcQ

5. Make sure you have ocw-content-rc populated with any course. For the sake of this example I'm using `6.004-spring-2017`.
6. `yarn start course`
7. Go to http://localhost:3000/youtube/
8. Observe that Hugo's default embedded YouTube player appears to play the video.

#### Any background context you want to provide?
Part of the issue was to check for ocw-studio-rc and ocw-studio data and update it such that it uses regular video resource instead of the prior, overriding embedded Youtube shortcode.
However, there does not appears to be any markdown data using the YouTube shortcode.

If we run inside shell:
```
from websites.models import WebsiteContent
wc = WebsiteContent.objects.filter(markdown__contains=" youtube ")
```
and print the contents of `wc`, we only get a single page for both ocw-studio-rc and ocw-studio:
https://ocw.mit.edu/courses/20-219-becoming-the-next-bill-nye-writing-and-hosting-the-educational-show-january-iap-2015/pages/student-reflections/joshua-cheongs-reflections/
If we run ctrl/cmd + f there, we will see that all of the times it is appearing there just as a regular youtube term.

And if we run:
`wc = WebsiteContent.objects.filter(markdown__contains="<youtube ")
`
In this case, `wc` is simply null.


#### Screenshots (if appropriate)
**Screenshot of the implemented change:**
<img width="1158" alt="image" src="https://user-images.githubusercontent.com/109785089/216296448-7096f736-a667-4a3d-95f5-55b98a501716.png">
